Lib/glyphNameFormatter/data/caseTable.txt

```diff
@@ -0,0 +1,1410 @@
+Case Folding Table
```

Comment:
there should be a `#` before Case Folding Table

Reply:
what should be the consensus be with double entries, like `I` `I` and `dotlessi`.

Reply:
this has been removed